docs/app/pages/components/components-sections/hierarchy-bar/hierarchy-bar/hierarchy-bar.component.html

```diff
@@ -41,7 +48,9 @@
         If specified, this node will be selected when the component is initialized.
     </tr>
 
-    <tr uxd-api-property name="mode" type="'standard' | 'collapsed'" defaultValue="standard">
+    <tr uxd-api-property name="mode"
+        type="'standard' | 'collapsed' | 'alternativeClickBehavior' | 'readOnly'"
```

Comment:
Perhaps `readOnly` should be `readonly` as this matches the casing used by the browser, for example `<input readonly>`

Reply:
I actually think that read-only should be a separate setting, in case we want to implement a read-only version of the standard layout in the future.